README.md

```diff
@@ -39,8 +46,19 @@ yarn               # or yarn install
 
 ### Alterar conteúdos
 
-Os ficheiros `.md` (markdown) estão na pasta `src`.
-Para alterar os links na sidebar, é necessário alterar o ficheiro `src/.vuepress/config.js`.
+Os ficheiros `.md` (markdown) estão na pasta da respetiva UC dentro de `content`.
+Assim que um ficheiro `.md` é adicionado à respetiva pasta da UC, é acessível a partir da sidebar.  
```

Comment:
This is not true, frontmatter metadata must be present (`path`, and optionally `type`)